cobbler/api.py

```diff
@@ -1498,15 +1498,18 @@ def signature_update(self):
 
     # ==========================================================================
 
-    def dump_vars(self, obj, formatted_output: bool = False):
+    def dump_vars(
+        self, obj, formatted_output: bool = False, remove_dicts: bool = False
+    ):
         """
         Dump all known variables related to that object.
 
         :param obj: The object for which the variables should be dumped.
         :param formatted_output: If True the values will align in one column and be pretty printed for cli example.
+        :param remove_dicts: If True the dictionaries will be put into str form.
```

Comment:
`remove_dict` sounds different than it's really supposed to do according to the descripton here.

Reply:
Yes Alex said this as well but I don't want to change the keyword usages in all of the codebase.